edge-hub/docker/windows/arm32v7/Dockerfile

```diff
@@ -8,7 +8,7 @@ WORKDIR /app
 
 COPY $EXE_DIR/ ./
 
-# Expose MQTT, AMQP and HTTPS ports
```

Comment:
I can revert this, but I thought this was better so when we add exposed ports we don't need to worry about the "and".